examples/large_models/inferentia2/inf2_handler.py

```diff
@@ -112,17 +111,27 @@ def inference(self, input_batch):
             list: A list of strings with the predicted values for each input text in the batch.
         """
         input_ids_batch, attention_mask_batch = input_batch
-        input_ids_batch = input_ids_batch
+
+        # insert padding if a partial batch was received
+        num_inferences = len(input_ids_batch)
+        logger.info("Input ids batch: %s", input_ids_batch)
+        logger.info("Num inferences: %s", num_inferences)
+        padding = self.batch_size - num_inferences
+        if padding > 0:
```

Comment:
Thansk @namannandan, I am wondering if it would be better to ask user to make sure register model with batch size consistent with what they have compiled the model with (adding this in README) instead of appending dummy inputs to the batch.

Reply:
@HamidShojanazeri Yes, that should work too. Although adding support for padding would make the model compatible with the batch aggregator in torchserve.

Reply:
I see, sure, lets then add some loggings to make user know when this is happening. I already updated the README as well.